vector/src/main/java/im/vector/features/hhs/ResourceLimitEventListener.kt

```diff
@@ -36,6 +36,11 @@ private const val MAX_PINNED_NOTICES_PER_ROOM = 2
 sealed class LimitResourceState {
     object Normal : LimitResourceState()
     data class Exceeded(val roomId: String, val eventId: String, val matrixError: MatrixError) : LimitResourceState()
+
+    fun softErrorOrNull(): MatrixError? = when (this) {
+        is Exceeded -> matrixError
+        is LimitResourceState.Normal -> null
```

Comment:
Remove `LimitResourceState.` and put normal first to keep the same order that the declaration